platform/migration/platform-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -48,10 +53,17 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
 
     @Override
     protected Object doExecute() {
-        outputInfoMessage(String.format(STARTING_EXPORT_MESSAGE, defaultExportDirectory));
+        Path exportDirectory;
+        if (exportDirectoryArgument == null || exportDirectoryArgument.isEmpty()) {
```

Comment:
Anybody knows what the convention is here? If the argument is `null`, does it mean that the parameter wasn't used and empty means that the argument was used but no value was provided? If empty means that no path was provided, shouldn't we just output some error message and return?